pod/perldelta.pod

```diff
@@ -45,6 +45,35 @@ XXX For a release on a stable branch, this section aspires to be:
 
 [ List each incompatible change as a =head2 entry ]
 
+=head2 Apostrophe is again recognized as a global name separator
+
+This was deprecated in Perl 5.38 and removed as scheduled in perl
+5.41.3, after some discussion has been reinstated by default.
```

Comment:
s/after/but after/ for better flow of thought.

Reply:
added